src/pages/ProgramsList/ProgramItems.js

```diff
@@ -56,36 +55,41 @@ for (let i = 0; i < 23; i++) {
 const ProgramItems = () => {
   const navigate = useNavigate();
 
-  const [currentTab, setCurrentTab] = useState("1");
-  const [items, setItems] = useState(dummyData);
-  const registeredPrograms = dummyData.slice(10, 16);
+  const [allPrograms, setAllPrograms] = useState();
+  useEffect(() => {
+    (async () => {
+      const programs = await Db.getAllPrograms();
+      setAllPrograms(programs);
+      setItems(programs);
+    })();
+  }, []);
 
-  const [filteredItems, setFilteredItems] = useState();
-  const [isFiltered, setIsFiltered] = useState(false);
+  // Selected programs according to the tabs "All Programs" vs "Registered Programs"
+  const [items, setItems] = useState();
+  // TODO: Filter registered programs
+  const registeredPrograms = allPrograms ? allPrograms.slice(1, 2) : {};
+  const [currentTab, setCurrentTab] = useState("1");
 
-  const handleClick = key => {
+  const handleClick = (key) => {
     setCurrentTab(key);
     if (key === "1") {
-      setItems(dummyData);
+      setItems(allPrograms);
     } else {
       setItems(registeredPrograms);
     }
   };
 
+  // Filtered programs according to the categories in the sidebar
+  const [filteredItems, setFilteredItems] = useState();
+  const [isFiltered, setIsFiltered] = useState(false);
+
   const [isModalVisible, setModalVisible] = useState(false);
   const [modalItem, setModalItem] = useState();
-  const showModal = item => {
+  const showModal = (item) => {
```

Comment:
Also, I think my prettier config adds brackets to single arguments for lambda functions 😓 Should we standardize the config to remove brackets instead?

Reply:
Yeap we should haha! If not our prettiers keep changing these stuff back and forth! I think we can just use a popular standard, I'm ok with anything!